### ISSUE TITLE
Cannot read properties of undefined (reading 'setAtlas')

### ISSUE THREAD

**Issue by VSCodeTriageBot** [microsoft/vscode#174053]
```javascript
TypeError: Cannot read properties of undefined (reading 'setAtlas')
at p._refreshCharAtlas in vscode-file://vscode-app/node_modules.asar/xterm-addon-webgl/lib/xterm-addon-webgl.js:1:20362
at p._handleOptionsChanged in vscode-file://vscode-app/node_modules.asar/xterm-addon-webgl/lib/xterm-addon-webgl.js:1:19062
at <anonymous> in vscode-file://vscode-app/node_modules.asar/xterm-addon-webgl/lib/xterm-addon-webgl.js:1:15765
at t.EventEmitter.fire in vscode-file://vscode-app/node_modules.asar/xterm/lib/xterm.js:1:139525
at a.i in vscode-file://vscode-app/node_modules.asar/xterm/lib/xterm.js:1:266845
at e.Terminal.s in vscode-file://vscode-app/node_modules.asar/xterm/lib/xterm.js:1:273006
at Ki.reuseTerminal in out/vs/workbench/workbench.desktop.main.js:2545:161
at async P.Fb in src/vs/workbench/contrib/tasks/browser/terminalTaskSystem.ts:1451:4
at async P.xb in src/vs/workbench/contrib/tasks/browser/terminalTaskSystem.ts:975:24
```
[Go to Errors Site](https://vscode-errors.azurewebsites.net/card?ch=e2816fe719a4026ffa1ee0189dc89bdfdbafb164&bH=025bfb53-d4ff-5cae-3b36-dd3467f75343)

**Comment by lorand-horvath** [comment]
I've been actually seeing this error about setAtlas pop up quite often...

**Comment by Tyriar** [comment]
This was reported in 1.75, already fixed in 1.76: https://github.com/microsoft/vscode/issues/172782

/duplicate
